mautic/mautic
# Lookup field for Lead title is properly visible now.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ x ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #12576 



#### Description:
Fixed the Lead lookup field for Title (Mr, Ms, etc.).


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Make sure you do not have a _localStorage_ data with the key like `__/s/ajax?action=lead:fieldList&field=title__data`
3. Try to create and edit contact. Focus the "title" field and try to type, it should auto-suggest, if you have typed at least two letters. 
4. Edit the sample data segment 'Like segment test with field percent sign at end' and check the behaviour of the auto suggest in the filter field for Title.
5. For more info refer the linked bug report
